react-hyf/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-hyf",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "install-server": "cd server && npm install",
+    "install-prev-client": "cd previousHomework && npm install",
+    "install-client": "cd react-hyf-client && npm install",
+    "install-all": "concurrently npm:install-server npm:install-client npm:install-prev-client",
+    "build-client": "cd react-hyf-client && npm run build",
+    "build-prev-client": "cd previousHomework && npm run build",
```

Comment:
Smart to separate your homework in this way and to be able to start it all at once using npm scripts 👏